### PR TITLE
Feature/generic node 10

### PR DESCRIPTION
**Pull request by csanquer**
* use official Node 10.15.1 as base image
* add docker multi-stage building
* add Makefile to easily build all version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,17 @@ Cubyn base docker container for Node.js, with yarn
...
+See [Docker Hub](https://hub.docker.com/r/cubyn/node/tags/) page for the list of available tags.
 
 ```sh
-docker build -t cubyn/node:{tag} .
+# building all images
+make build
+# tagging all image versions
+make tag
+# pushing all image versions
+make push
```
Comment (FPierre) [inline README.md]: Possible to add a new Node version buid "scenario"? (all commands required to create new Docker image from a new Node version)

File: generic/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# For CI Builds ONLY
+# tags are cubyn/node:ci-{version}
+FROM base as ci
+
+RUN apk update && apk add --no-cache \
+   python \
+   g++ \
+   libexecinfo-dev \
+   mysql-client
```
Comment (getvega) [inline generic/Dockerfile]: I think mysql-client is needed in our production images as well (req to run migrations)

Comment (csanquer) [reply generic/Dockerfile]: Sure ?

File: README.md (hunk 1 above)
Comment (csanquer) [reply README.md]: I don't understand your question.

Comment (csanquer) [reply README.md]: resolved by README improvement

**Pull request merged**